src/middlewares/logger-middleware.ts

```diff
@@ -1,92 +1,65 @@
-import * as graphqlFields from 'graphql-fields';
 import { inject, injectable } from 'inversify';
 import { POLARIS_TYPES } from '../inversion-of-control/polaris-types';
 import { GraphqlLogProperties } from '../logging/graphql-log-properties';
 import { GraphqlLogger } from '../logging/graphql-logger';
 import { PolarisContext } from '../server/polaris-context';
-import { MiddlewareParams, PolarisMiddleware, PostMiddlewareParams } from './polaris-middleware';
+import {
+    PolarisMiddleware,
+    RequestMiddlewareParams,
+    ResponseMiddlewareParams,
+} from './polaris-middleware';
 
 @injectable()
 export class LoggerMiddleware<TContext extends PolarisContext = PolarisContext>
     implements PolarisMiddleware<TContext> {
+    static buildProps(context: any): GraphqlLogProperties {
+        return {
+            operationName: context.body.operationName,
+            request: {
+                requestQuery: {
+                    body: context.body.query,
+                },
+            },
+        };
+    }
+
     @inject(POLARIS_TYPES.GraphqlLogger) polarisLogger!: GraphqlLogger;
 
-    preResolve({ root, args, context, info }: MiddlewareParams<TContext>) {
+    preResolve({ root, args, context, info }: RequestMiddlewareParams<TContext>) {
+        const props: GraphqlLogProperties = LoggerMiddleware.buildProps(context);
         if (!root) {
-            const props: GraphqlLogProperties = this.initReqProperties({
-                root,
-                args,
-                context,
-                info,
-            });
             this.polarisLogger.debug(
-                `Resolver of ${props.operationName}
-             began execution. Arguments received: ${props.request &&
-                 props.request.requestQuery &&
-                 props.request.requestQuery.body}`,
+                `Resolver of ${props.operationName} began execution. Action is:
+                    \n${context.body.query}Arguments given:${JSON.stringify(args)}`,
                 {
                     context,
                     polarisLogProperties: props,
                 },
             );
         } else {
-            const props: GraphqlLogProperties = this.initReqProperties({
-                root,
-                args,
-                context,
-                info,
-            });
-            this.polarisLogger.debug(`field ${info.fieldName} began execution.`, {
+            this.polarisLogger.debug(`field fetching of ${info.fieldName} began execution.`, {
                 context,
                 polarisLogProperties: props,
```

Comment:
remove ","
the prefix polarisLogProperties is required? can you apply the same standard to the whole object?
Same in every log